files/dcgmd.conf

```diff
@@ -1,6 +1,6 @@
 # This file is managed by Puppet, manual changes will be lost
 [[inputs.exec]]
-  command = "/etc/telegraf/scripts/dcgmi_stats.sh"
+  command = "/etc/telegraf/scripts/dcgm/dcgmi_stats.sh"
```

Comment:
This path in the telegraf config needed to be updated to match the script's path that is on line 33 in profile_gpu::dcgm::telegraf (manifests/dcgm/telegraf.pp).